dags/openshift_nightlies/util/executor.py

```diff
@@ -21,14 +21,14 @@ def get_default_executor_config():
             )
         }
 
-def get_executor_config_with_cluster_access(release: OpenshiftRelease):
+def get_executor_config_with_cluster_access(release: OpenshiftRelease, executor_image='quay.io/keithwhitley4/airflow-ansible:2.1.0'):
```

Comment:
@whitleykeith thoughts on moving your image to the cloud-bulldozer quay org?

Reply:
I'm working on [this branch](https://github.com/whitleykeith/airflow-kubernetes/tree/ci) to add a CI framework to Airflow which would include doing that